### PR TITLE
💥 Add Sei Arctic Testnet

### PR DESCRIPTION
**Pull request by dangerousfood**
As title. Some documentation can be found [here](https://v2.docs.sei.io/overview).

### CODE REVIEW

File: src/networks.ts [hunk 1]
```diff
@@ -108,6 +109,7 @@ export const explorers = [
   "https://holesky.fraxscan.com/",
   "https://sepolia-explorer.metisdevops.link/",
   "https://sepolia.explorer.mode.network/",
+  "https://evm-rpc-arctic-1.sei-apis.com",
```
Comment (pcaversaccio) [inline src/networks.ts:112]: This is an RPC; it requires a block explorer link here...

**Review by pcaversaccio — COMMENTED** [review]
`xdeployer` is powered by my [`CreateX`](https://github.com/pcaversaccio/createx) factory. Can you please fund my deployer account `0xeD456e05CaAb11d66C4c797dD6c1D6f9A7F352b5` so I can deploy it on this testnet? I will review first however, if the EVM version is equivalent for the opcodes which my factory uses ([reference](https://v2.docs.sei.io/differences-with-ethereum)).

File: src/networks.ts (hunk 1 above)
Comment (dangerousfood) [reply src/networks.ts]: ok, I have fixed this

**Comment by dangerousfood** [comment]
> `xdeployer` is powered by my [`CreateX`](https://github.com/pcaversaccio/createx) factory. Can you please fund my deployer account `0xeD456e05CaAb11d66C4c797dD6c1D6f9A7F352b5` so I can deploy it on this testnet? I will review first however, if the EVM version is equivalent for the opcodes which my factory uses ([reference](https://v2.docs.sei.io/differences-with-ethereum)).

Wallet is now funded

https://seistream.app/transactions/0x514f6a9c352c7e42aac38f86d293d394048711569a834e91e5964140a7b2f4dd

**Comment by pcaversaccio** [comment]
> > `xdeployer` is powered by my [`CreateX`](https://github.com/pcaversaccio/createx) factory. Can you please fund my deployer account `0xeD456e05CaAb11d66C4c797dD6c1D6f9A7F352b5` so I can deploy it on this testnet? I will review first however, if the EVM version is equivalent for the opcodes which my factory uses ([reference](https://v2.docs.sei.io/differences-with-ethereum)).
> 
> Wallet is now funded
> 
> https://seistream.app/transactions/0x514f6a9c352c7e42aac38f86d293d394048711569a834e91e5964140a7b2f4dd

Can you quickly confirm to me that `CREATE` and `CREATE2` opcodes work the same as on any EVM equivalent chain?

**Comment by pcaversaccio** [comment]
Deployed: https://github.com/pcaversaccio/createx/pull/110.

**Review by pcaversaccio — APPROVED** [review]
Quickly adjusted the internal logic since the block explorer links for Sei are not the same as for Etherscan and Blockscout. Tested and it works great:

image

Do you need a new release of `xdeployer` this week already or can you wait until next week?

**Comment by pcaversaccio** [comment]
Released via [`v3.0.9`](https://github.com/pcaversaccio/xdeployer/releases/tag/v3.0.9).

**Pull request merged**